schema/geometry_schema.json

```diff
@@ -930,7 +931,8 @@
 					"minimum": 0,
 					"exclusiveMinimum": true,
 					"si_units": "m",
-					"ip_units": "ft"
+					"ip_units": "ft",
+					"default": 4
```

Comment:
is this why you were talking about trailing commas earlier?

Reply:
yep, that's exactly why. Notice, `line 935`. When webpack reports an error it gives the number of *characters* into the file where the problem occurs.